README.md

```diff
@@ -43,6 +46,33 @@ impl std::fmt::Display for MyError {
 }
 ```
 
+If you only have a single error case you can also generate a struct:
+```rust
+extern crate custom_error;
+use custom_error::custom_error;
+
+// Note the use of braces rather than parentheses.
+custom_error!{MyError{code:u8} = "error with code {code}."}
+```
+
+instead of
```

Comment:
I don't think we need to reproduce the whole expanded code a second time. People get the idea.